WooCommerce/Classes/POS/Card Present Payments/CardPresentPaymentEvent.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+enum CardPresentPaymentEvent {
+    case idle
+    case showAlert(_ alertViewModel: CardPresentPaymentsModalViewModel)
+    case showReaderList(_ readerIDs: [String], selectionHandler: ((String) -> Void))
+    case showOnboarding(_ onboardingViewModel: InPersonPaymentsViewModel)
```

Comment:
Similar to above, but I think this one is less in need of replacing/translating.

Reply:
I'm taking on the renaming https://github.com/woocommerce/woocommerce-ios/issues/12863, let's merge this first and I'll pull the latest trunk after.